Standalone/Standalone/NativeSimpleViewController.h

```diff
@@ -0,0 +1,31 @@
+//
+//  NativeSimpleViewController.h
+//  Standalone
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <UIKit/UIKit.h>
+#import "CriteoNativeAdView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NativeSimpleViewController : UIViewController
+
+@property (weak, nonatomic) IBOutlet UIView *nativePlaceholder;
```

Comment:
Is this property accessible outside the class? Otherwise, private properties should be defined in `.m` file

Reply:
Moved to `NativeSimpleViewController.m`.